### DISCUSSION TITLE
the html first POP RTA  confirm window,can't save real wanted content by singlefile under window CMD mode.

### DISCUSSION THREAD

**Discussion by aw894** [gildas-lormeau/single-file-cli#142] [Q&A]
the html first POP RTA  confirm window,can't save real wanted content by singlefile under window CMD mode
How can I resolve tjhe problem? Thanks!

![AAAAA](https://github.com/user-attachments/assets/942bf0bc-ffd2-4c3b-8e3d-756683d782c4)

**Comment by gildas-lormeau** [comment]
You could use your existing Chrome user profile by passing the option `--browser-arg --user-data-dir=...` where ... is the path to your Chrome user profile folder

**Reply by aw894** [reply]
Thanks for you
